crates/starknet-os/src/utils.rs

```diff
@@ -129,7 +129,7 @@ fn get_tokio_runtime_handle() -> Result<tokio::runtime::Handle, HintError> {
 
 /// Executes a coroutine from a synchronous context.
 /// Fails if no Tokio runtime is present.
-pub(crate) fn execute_coroutine<F, T>(coroutine: F) -> Result<T, HintError>
+pub fn execute_coroutine<F, T>(coroutine: F) -> Result<T, HintError>
```

Comment:
I've been thinking that we need a `utils` crate in the workspace for this type of things, this should not be a public function of the OS library itself. But it could be a public function of a `starknet-os-utils` crate in the workspace.